method/application/get.js

```diff
@@ -0,0 +1,63 @@
+const pull = require('pull-stream')
+module.exports = function GroupApplicationGet (server) {
+  return function groupApplicationGet (applicationId, cb) {
+    const query = [
+      {
+        $filter: {
+          dest: applicationId
+        }
+      }
+    ]
+
+    server.get({ id: applicationId, private: true }, (rootErr, rootData) => {
+      if (rootErr) cb(rootErr)
```

Comment:
this needs to be `return cb(rootErr)` otherwise it will continue on with the pull!

I would also put a check in here to check `isValid(rootData)`

Reply:
Good call!